src/hooks/screen/useChatScreenHook.ts

```diff
@@ -153,15 +153,17 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
             channelList.description = message;
             channelList.lastUpdatedBy = userId;
             channelList.lastUpdatedAt = new Date().toISOString();
-            channelList.save(localDb);
+            await channelList.save(localDb);
           }
         }
 
         refresh('chat');
         refresh('channelList');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The save operation on the `channelList` object is now awaited using `await`. This ensures that the save operation completes before the code execution proceeds, which can prevent potential issues related to asynchronous operations. The refresh calls are also moved inside the if block, ensuring they only execute when a save operation occurs.
```diff
-            channelList.save(localDb);
-         }
-         refresh('chat');
-         refresh('channelList');
+            await channelList.save(localDb);
+            refresh('chat');
+            refresh('channelList');
+         }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->